package.json

```diff
@@ -75,6 +75,7 @@
   "scripts": {
     "build": "gulp build",
     "lint": "gulp lint sass-lint",
+    "package": "cd addon && npm run package",
```

Comment:
Not sure of the best solution. If the command is only intended for local development, it probably isn't a big deal since we'd know that we need to run `npm install` in both places. If we're trying to be a bit more friendly, we could possibly do something a bit hackier like:
```js
"package": "cd addon/ && npm install && npm run package",
```

Or, we could potentially do some super hacky-ness and add a "postinstall" solution to the root /package.json which does the CD into ./addon/ and runs `npm install`. But maybe that's getting too meta. But we have used that trick a few times in the past to auto-install Bower deps after you run `npm install`.

```js
"package": "cd addon/ && npm run package",
"postinstall": "cd addon/ && npm install",
```

Or maybe even add a "prepackage" task which does `cd addon/ && npm install`, so it only attempts to install those deps if you specifically run `npm run package`.

But I'm definitely open to other suggestions from 6a68/Chuck/Les/Wil/anybody.